packages/kit/test/apps/options-2/src/routes/ws/+server.js

```diff
@@ -0,0 +1,39 @@
+let sockets = [];
+
+export const socket = {
```

Comment:
```suggestion
/** @type {import("crossws").Hooks} */
export const socket = {
```
Should we re-export the `Hooks` type provided by `crossws` or rely on their types here?

Reply:
I was hoping we could actually type it automatically the same way the GET functions are automatically typed when exported from the same file, I would love to handle that for the users.

I think Ben mentioned that @dummdidumm was the typing expert here, they maybe be able to provide some insight on the best way to get this the last mile here.

Reply:
I think the automatic typing is part of the Svelte language tools VS Code extension. So we would still need the manual typing available here.

Reply:
I've re-exported the `Hooks` type as `Socket` but I'm wondering if we should also re-export `Peer` and `Message` as they seem important/useful. https://github.com/unjs/crossws/blob/bc55c9765f436316213e9a3b907522cc86013a8c/src/index.ts#L9-L13
cc: @pi0